slices_demo1/README.md

```diff
@@ -108,17 +120,29 @@ control egress {
 }
 ```
 
+**modification**:
+
 **改动：**
 
-1.增加一个寄存器`Register_Count`，并设置为direct状态。
+1.Add a register `Register_Count` and set it to the **direct** state.
 
-2.增加一张名为Count_Table的流表，流表内含动作`_drop`与`Read_Register`。该流表根据端口进行匹配，通过command.txt(或者Runtime_CLI)增加一条表项：倘若字段easyroute_port与3相匹配，执行动作`Read_Register`；同时设置默认动作`_drop`，当没有匹配到表项时执行丢包。
+  增加一个寄存器`Register_Count`，并设置为direct状态。
 
-3.动作`Read_Register`：将寄存器中的信息记录到元数据中。
+2.Add a flow table named `Count_Table`, the flow table contains the action `_drop` and `Read_Register`. The flow table matches the port，and add a flow entry by **command.txt** (or **Runtime_CLI**) : if the field `easyroute_port` matches 3, the action `Read_Register` is executed; the default aciton `_drop` is set ,  the packet is droped when no entry is  matched.
```

Comment:
contains => includes;

"The flow table matches the port，and add a flow entry by **command.txt** (or **Runtime_CLI**) ..."  => 

The incoming packets are matched by its port number. We deployed the runtime commands to the switch so that the switch can generate flow entries to execute actions for matched packets. The action "Read_Register" executed as soon as the field `easyroute_port` of the incoming packet was the port 3. The switch drop the packet which is the default action of the flow table if the packet didn't find its matched entry.